ivy/functional/frontends/numpy/matrix/methods.py

```diff
@@ -117,3 +118,13 @@ def any(self, axis=None, out=None):
         if ivy.exists(axis):
             return any(self.A, axis=axis, keepdims=True, out=out)
         return any(self.A, axis=axis, out=out)
+
+    def sum(self, axis=None, dtype=None, out=None, keepdims=False, initial=0, where=True):
+        """Return the sum of the array elements over the given axis."""
+        if not ivy.exists(axis):
+            axis = None
+
+        return sum(
+                self.A, axis=axis, dtype=dtype, out=out, keepdims=keepdims, initial=initial,
```

Comment:
please also update the parameters here accordingly as per the required arguments of the method